app/jobs/clean_all_problems_job.rb

```diff
@@ -0,0 +1,8 @@
+class CleanAllProblemsJob < ActiveJob::Base
```

Comment:
Or I guess if the convention is to append Job, it could be `DestroyProblemsByAppJob`